invenio_requests/assets/semantic-ui/js/invenio_requests/contrib/labels/StatusLabel.js

```diff
@@ -9,59 +9,53 @@ import { Label, Icon } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_requests/i18next";
 
 export const LabelStatusSubmit = (props) => {
```

Comment:
Minor: props can be removed. Also applies for the rest.
```suggestion
export const LabelStatusSubmit = (props) => {
```